deployments/kubernetes/charts/clowder2/values.yaml

```diff
@@ -168,25 +168,31 @@ rabbitmq:
     #storageClass: csi-cinder-sc-retain
 
 # ----------------------------------------------------------------------
-# EXTRACTORS
+# DEFAULT EXTRACTOR SERVICES
 # ----------------------------------------------------------------------
+message:
+  image:
+    repository: clowder/clowder2-messages
+    tag: main
```

Comment:
better to comment these out, otherwise it will not default to the chart.appversion in your templates by defaul. Best to have people set this in the overrides.